### DISCUSSION TITLE
Thank you, @grahampugh

### DISCUSSION THREAD

**Discussion by dan-snelson** [grahampugh/erase-install#74] [General]
# Thank you, @grahampugh!

Graham:

Thanks for an excellent, continuously updated mission-critical tool and for being such a great example of helping others succeed.

You inspire me.

**Comment by grahampugh** [comment]
Thanks Dan, appreciate the nice feedback!

**Comment by julienvs** [comment]
Hi @grahampugh ,

I was looking for a place where I could thank you and found this thread.
Fast to deploy and great user experiance THANKS-A-LOT!

Julien

**Comment by pauletello** [comment]
Honestly, thank you so much for this script. 
I am currently testing it for JAMF at my work. From the few machines I have tested it on, it works fantastic. 

I appreciate all the work that went into this. 

I used the pkg method and the execution command and had no issues to report.

**Comment by thdenton** [comment]
Thanks @grahampugh we have starting using you script recently. We have around 800 machines to update this summer to Monterey. It working great so far.
